.travis/build.sh

```diff
@@ -30,5 +30,13 @@ export NRF5_TOOLS_ROOT=${TRAVIS_BUILD_DIR}/nRF5x-Command-Line-Tools
 # Export GNU_INSTALL_ROOT variable pointing to the ARM GCC tool chain.
 export GNU_INSTALL_ROOT=${TRAVIS_BUILD_DIR}/arm/gcc-arm-none-eabi-7-2018-q2-update/bin/
 
+if test ${TRAVIS_PULL_REQUEST} = "false"; then
+    VERSION_STRING=1.0d${TRAVIS_BUILD_NUMBER}
+else
+    VERSION_STRING=1.0d-CI-${TRAVIS_JOB_NUMBER}
```

Comment:
@jaylogue, thoughts ? 

Pull request builds are not deployed so this version string does not matter at the moment. But if in future we do deployment of PR builds then is the naming okay ?

Note: Travis CI branch builds get deployed and maybe we want some version naming differentiation in those from master.
For eg. 
1. When a branch is created and pushed, the CI branch build version will be something like 1.0d163. 
2. When a PR for that branch is created, the CI Pull Request build version will be something like 1.0d-CI-164
3. once the PR has been merged, next master CI build version will be something like 1.0d165
since 1 and 3 are deployed, there is no currently no way to distinguish if the build was a master build or branch build. 

We have two options:
1. Disable deployment of branch builds all together.
2. Add a condition in the build.sh script to check if the build is a branch build and if it is, add a variant to the version string.